docs/contributing.rst

```diff
@@ -62,13 +46,9 @@ Prepare Resolwe for development::
 Preparing database
 ==================
 
-Create a ``resolwe`` database::
-
-    # Remove database if exists
-    dropdb resolwe
+Start Docker containers::
```

Comment:
Docker containers should be started from "tests" subdirectory. Or with command 

> docker-compose --file tests/docker-compose.yml up --detach

Reply:
Thanks, I've moved `cd tests` few lines up.